scripts/src/test.sh

```diff
@@ -24,6 +24,7 @@ test_usage() {
     runtime-upgrade run runtime-upgrade test for queue processing
```

Comment:
GitHub diff shows empty `run_all_benchmarks.sh` file now

Reply:
I think it's gh issue, because the file exists in the branch with no changes. Seems `empty diff file`